ccdaservice/oe-blue-button-generate/lib/sectionLevel2.js

```diff
@@ -654,21 +660,22 @@ exports.healthConcernSection = function (htmlHeader, na) {
                 }, {
```

Comment:
@sjpadgett  Looks like there is a bug on line 659.  Should be
```
existsWhen: condition.keyDoesntExist("text")
```
Otherwise it fails validation as two text elements are being outputted.